notebooks/running_jobs.ipynb

```diff
@@ -169,14 +223,77 @@
     "config = read_config_file(\"test_config.txt\")\n",
     "outcfg = execute(**config)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Pipeline Stages\n",
+    "\n",
+    "The evcouplings pipeline is split into different stages for different aspects of computation. This allows users to run only the desired stages, and to re-start computation from an arbitrary stage. The stages are as follows:\n",
+    "\n",
+    "__align__: Creates sequence alignments or reads in an existing alignment, and pre-processes alignment for couplings calculation.\n",
+    "\n",
+    "__couplings__: Calculates evolutionary couplings, post-processes output into correct numbering, and fits statistical scoring model to couplings. \n",
+    "\n",
+    "__compare__: Searches for structures against which to compare the couplings, visualizes couplings in a contact map, calculates precision of couplings against structure. \n",
+    "\n",
+    "__mutate__: Generates and visualizes matrices of the predicted effects of mutations at every site. \n",
```

Comment:
would also add that you can predict a user-supplied list of mutations (any order) here too